gc_tests/tests/multithreaded.rs

```diff
@@ -20,11 +20,17 @@ impl Drop for PanicOnDrop {
 
 }
 
+// This can race, but it's benign because it's only read from the main thread,
+// and we only care that _at some point_ it gets flipped.
+static mut NO_CHILD_EXISTS: bool = true;
```

Comment:
Let's make this an `AtomicBool` with `Ordering::Relaxed` and then I think we're actually OK.

Reply:
Ok, I think it's probably safer to use Sequential Consistency because I don't want the load in the while loop to be reordered below the call to `force_gc`.

Reply:
Good point!

Reply:
Try feba5edaee7ee55a8824dcd18da8016fe20d41c1